src/tokens/search_env.c

```diff
@@ -102,13 +127,17 @@ int	search_env(t_env_lst *env, char **str)
 	return (EXIT_SUCCESS);
 }
 
-/* Function parse a str_lst node and expand it via search_env
-after doing that with every node copy it without the quotes
-separate between two tokens on special case 
-if separated, new token will need to be re-classifyied 
-command token will become command + argument 
-argument token will become 2 arguments 
-will need to make a function to do that with every tokens*/
+/**
+ * @brief Function parse a str_lst node and expand it via search_env.
+ * Copies node without the quotes.
+ * Separate between two tokens if space in an unquoted str_lst node.
+ * if separated, new token will need automatically be classifyed as T_ARGUMENT.
+ * 
+ * @param env_lst list containing the env variables.
+ * @param str_lst list with each part to expand.
+ * @param token node being expanded.
+ * @return EXIT_FAILURE or EXIT_SUCCESS. 
+ */
 
```

Comment:
Do not separate the function and its comment with an empty line. Doing so prevents Intellisense to get the comment's data and display it to us when hovering the function with the mouse anywhere else in the project.